.gitignore

```diff
@@ -4,6 +4,8 @@ dist
 
 # Ignore config files
 /config/*
+/ux/*
```

Comment:
Anyways I think we should apply this change, if I start a new project from the seed maybe I have to change some of the ux files and those changes should not be ignored.